packages/beacon-node/src/chain/chain.ts

```diff
@@ -284,6 +285,10 @@ export class BeaconChain implements IBeaconChain {
     await this.bls.close();
   }
 
+  isProcessingCurrentSlotBlock(): boolean {
+    return this._isProcessingCurrentSlotBlock;
+  }
```

Comment:
Should this logic and `_isProcessingCurrentSlotBlock` be in the network processor class? It knows the block's slot and current clock